src/main/java/seedu/address/logic/commands/client/DeleteClientCommand.java

```diff
@@ -69,14 +69,11 @@ public CommandResult execute(Model model) throws CommandException {
         // Do not delete Client unless user use the force flag
         if (model.hasAnyScheduleAssociatedWithClient(clientToDelete) && !isForced) {
             return new CommandResult(MESSAGE_FORCE_DELETE_CLIENT_USAGE);
+        } else {
```

Comment:
I think it is better to not have `else` after a return statement. It is following [Make the happy path prominent](https://nus-cs2103-ay2021s1.github.io/website/se-book-adapted/chapters/codeQuality.html#make-the-happy-path-prominent)

Reply:
oh @kelvinvin 's suggestion is to have the else to make it clearer
so make it like the prev commit?